ome/baseline_validators.go

```diff
@@ -6,8 +6,7 @@ import (
 	"strings"
 	"terraform-provider-ome/clients"
 
-	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
-	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
 )
 
 type outputFormatValidator struct {
```

Comment:
Looks to me like we do not actually need this custom validator anymore... Looks like [stringvalidator.OneOf](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework-validators@v0.9.0/stringvalidator#OneOf) is what can easily do the trick.

Reply:
Most of the Validators were in development when 0.0.10, a lot can be removed to use standards

Reply:
Thanks @rounak-adhikary and @sachin-apa ,
will be adding the OneOf validator in place of the custom validator.

Reply:
closing the comment, as we have added the string validator